main/signers/hot/HotSigner/worker.js

```diff
@@ -51,7 +43,7 @@ class HotSignerWorker {
     const signed = ecsign(hash, key)
 
     // Return serialized signed message
-    const hex = Buffer.concat([Buffer.from(signed.r), Buffer.from(signed.s), Buffer.from([signed.v])]).toString('hex')
+    const hex = Buffer.concat([Buffer.from(signed.r), Buffer.from(signed.s), Buffer.from([Number(signed.v)])]).toString('hex')
```

Comment:
`r` and `s` are already `Buffer`s so I don't think we need to wrap them again. is `Number(bigint)` a safe operation? seems like there could be an overflow